cli/index.ts

```diff
@@ -429,12 +452,26 @@ function getPostgresCredentials() {
 
 function getJdbcCredentials(hostQuestion: string, defaultPort: number): dataform.IJDBC {
   const host = readlineSync.question(commandOutput(hostQuestion));
-  const port = readlineSync.questionInt(
-    commandOutput(`Enter the port that Dataform should connect to (usually ${defaultPort}):\n`)
+  writeStdOut(
+    commandOutput(`Enter the port that Dataform should connect to (leave blank to use default):`)
+  );
+  const port = parseInt(
+    readlineSync.prompt({
+      limit: value => {
+        const intValue = parseInt(value);
+        return (
+          !isNaN(intValue) && typeof intValue === "number" && intValue >= 0 && intValue <= 65536
```

Comment:
perhaps just use `Number.isInteger(intValue)` instead of the first two conditions? (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger)

Reply:
Good idea. Sadly: if I pass in `value`, typescript complains I'm not passing in a `number` - so I could pass `intValue`, but we already know that's going to be an integer...

Reply:
`parseInt(null)` returns `NaN` which is not an integer. Any non number string would return `NaN`.

The point I was making was that `Number.isInteger` means you can get rid of the first two conditions.

Investigating more, I actually think the `typeof intValue === "number"` is irrelevant since the only two possible values of `parseInt` are an integer or `NaN`. And `typeof NaN` evaluates to "number" 🙃 

So actually, I would say, just drop the typeof condition and then leave things as they are.

Reply:
Thanks, done